src/org/omegat/util/gui/FontFallbackManager.java

```diff
@@ -52,15 +57,117 @@ private FontFallbackManager() {
      * <li><a href="https://github.com/googlei18n/noto-emoji">Noto Emoji</a>
      * </ul>
      */
-    private static final Set<String> FONT_BLACKLIST = Collections.singleton("Apple Color Emoji");
-    private static final Font FONT_UNAVAILABLE = new Font("", 0, 0);
+    private static final Set<String> FONT_BLACKLIST = new HashSet<>(
+            Arrays.asList("Apple Color Emoji", "Noto Color Emoji")
```

Comment:
Did you actually test Noto Color Emoji, or are you basing this on my comment on SourceForge?

Reply:
I tested on Linux.